src/templates/filtersDrawer.jsx

```diff
@@ -44,6 +55,13 @@ const propTypes = {
             checkbox: PropTypes.shape({
                 checked: PropTypes.bool,
                 className: PropTypes.string,
+                /**
+                 * A DatePickerInput can be disabled.
```

Comment:
Should be "A Checkbox can be disabled."  (Or maybe "A FiltersDrawer Row Item Checkbox can be disabled.")

Reply:
Fixing...

Reply:
Commit 7c8b691 fixes this.

Reply:
Looks good.  Resolving.